Formula/polymake.rb

```diff
@@ -0,0 +1,56 @@
+class Polymake < Formula
+  desc "Tool for computations in algorithmic discrete geometry"
+  homepage "https://polymake.org"
+  url "https://polymake.org/lib/exe/fetch.php/download/polymake-3.2r3.tar.bz2"
+  version "3.2"
```

Comment:
Version number should be autodetected, remove this line.

Reply:
line deleted

Reply:
Now only 3 is detected, not 3.2.